tests/integration/indicator-network-test-base.cpp

```diff
@@ -604,6 +605,29 @@ QString IndicatorNetworkTestBase::createVpnConnection(const QString& id,
     return path.path();
 }
 
+QString IndicatorNetworkTestBase::importVpnConnection()
+{
+    nmofono::connection::ActiveConnectionManager::SPtr actConnManager =
+        make_shared<nmofono::connection::ActiveConnectionManager>(
+            QDBusConnection::systemBus()
+        );
+    nmofono::vpn::VpnManager::SPtr vpnManager = make_shared<nmofono::vpn::VpnManager>(
+        actConnManager,
+        QDBusConnection::systemBus()
+    );
+    const QString& ovpn_sample("sample.ovpn");
+
+    auto reply = vpnManager->importConnection(
+        nmofono::vpn::VpnConnection::Type::openvpn,
+        ovpn_sample
+    );
+    if (reply.isNull())
+    {
+        EXPECT_FALSE("Connection import failed");
+    }
```

Comment:
This block doesn't really make any sense. `EXPECT_FALSE()` expects a boolean expression. So what are you trying to do here? I guess there's also no need to check reply here, as it's still being returned anyway, and the test won't stop processing.

Reply:
My bad, I copied wrongly from the examples around, testing is always a bit daunting to me, so these kind of oddities comes out of my mind